### PR TITLE
vstart: out dir can contain a large number of files

### PR DESCRIPTION
**Pull request by mgfritch**
Running an aggressive test against a long running vstart environment managed to create a very large number of log files in the output dir.

```
$ ../src/vstart.sh -n --cephadm
<snip>
rm -f core* 
../src/vstart.sh: line 1208: /usr/bin/rm: Argument list too long

$ ls out/* | wc -c
799119
```

Signed-off-by: Michael Fritch <mfritch@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/vstart.sh [hunk 1]
```diff
@@ -1205,7 +1205,7 @@ fi
 [ -d $CEPH_OUT_DIR  ] || mkdir -p $CEPH_OUT_DIR
 [ -d $CEPH_DEV_DIR  ] || mkdir -p $CEPH_DEV_DIR
 if [ $inc_osd_num -eq 0 ]; then
-    $SUDO rm -rf $CEPH_OUT_DIR/*
+    $SUDO find "$CEPH_OUT_DIR" | xargs rm -rf
```
Comment (batrick) [inline src/vstart.sh:1208]: better would be

`$SUDO find "$CEPH_OUT_DIR" -type f -delete`

**Comment by mgfritch** [comment]
jenkins test make check

**Comment by mgfritch** [comment]
ping @batrick

**Comment by mgfritch** [comment]
retest this please

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2020-07-18_13:35:09-rados-wip-kefu-testing-2020-07-18-1927-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2020-07-19_05:50:32-rados-wip-kefu-testing-2020-07-18-1927-distro-basic-smithi/

failures tracked by

- https://tracker.ceph.com/issues/44990
- https://tracker.ceph.com/issues/45721

**Pull request merged**